access_custom_page.go

```diff
@@ -0,0 +1,104 @@
+package cloudflare
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+type AccessCustomPageType string
+
+const (
+	Forbidden      AccessCustomPageType = "forbidden"
+	IdentityDenied AccessCustomPageType = "identity_denied"
+)
+
+type AccessCustomPage struct {
+	// The HTML content of the custom page.
+	CustomHTML string               `json:"custom_html,omitempty"`
+	Name       string               `json:"name,omitempty"`
+	AppCount   int                  `json:"app_count,omitempty"`
+	Type       AccessCustomPageType `json:"type,omitempty"`
+	UID        string               `json:"uid,omitempty"`
+}
+
+type AccessCustomPageListResponse struct {
+	Response
+	Result     []AccessCustomPage `json:"result"`
+	ResultInfo `json:"result_info"`
+}
+
+type AccessCustomPageResponse struct {
+	Response
+	Result AccessCustomPage `json:"result"`
+}
+
+func (api *API) AccessCustomPages(ctx context.Context, rc *ResourceContainer, pageOpts PaginationOptions) ([]AccessCustomPage, error) {
```

Comment:
we should prefix this method with `List` but also update it to match the newer client conventions of having the dedicated third parameter for searching.

example: https://github.com/cloudflare/cloudflare-go/blob/master/rulesets.go#L750 
more docs: https://github.com/cloudflare/cloudflare-go/blob/master/docs/experimental.md#consistent-crud-method-signatures

```suggestion
func (api *API) ListAccessCustomPages(ctx context.Context, rc *ResourceContainer, pageOpts PaginationOptions) ([]AccessCustomPage, error) {
```